packages/table/src/table.tsx

```diff
@@ -530,14 +556,26 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     }
 
     private bodyCellRenderer = (rowIndex: number, columnIndex: number) => {
-        return this.getColumnProps(columnIndex).renderCell(rowIndex, columnIndex);
+        const columnProps = this.getColumnProps(columnIndex);
+        const cell = columnProps.renderCell(rowIndex, columnIndex);
+        const cellLoading = cell.props.loading;
+
+        let loading: boolean;
+        if (cellLoading != null) {
+            loading = cellLoading;
+        } else {
+            loading = this.hasLoadingOption(columnProps.loadingOptions, ColumnLoadingOption.CELLS);
+        }
```

Comment:
i tend to prefer ternary ifs for this pattern so you can define the variable as a `const` and immediately assign the correct value.

```js
const loading = (cellLoading == null)
    ? this.hasLoadingOption(columnProps.loadingOptions, ColumnLoadingOption.CELLS)
    : cellLoading; 
```